modules/swagger-parser-v3/src/test/java/io/swagger/parser/v3/util/OpenAPIDeserializerTest.java

```diff
@@ -216,6 +216,38 @@ public void readServerObject(JsonNode rootNode) throws Exception {
 
     }
 
+    @Test
+    public void readMissingServerObject() throws Exception {
+        final ObjectMapper mapper = new ObjectMapper(new YAMLFactory());
+        final JsonNode rootNode = mapper.readTree(Files.readAllBytes(java.nio.file.Paths.get(getClass().getResource("/oas.yaml").toURI())));
+
+        final OpenAPIDeserializer deserializer = new OpenAPIDeserializer();
+        final SwaggerParseResult result = deserializer.deserialize(rootNode);
+
+        Assert.assertNotNull(result);
+
+        final OpenAPI openAPI = result.getOpenAPI();
+        Assert.assertNotNull(openAPI);
+
+        assertEquals(openAPI.getServers().get(0).getUrl(),"/");
+    }
+
+    @Test
+    public void readEmptyServerObject() throws Exception {
+        final ObjectMapper mapper = new ObjectMapper(new YAMLFactory());
+        final JsonNode rootNode = mapper.readTree(Files.readAllBytes(java.nio.file.Paths.get(getClass().getResource("/oas2.yaml").toURI())));
+
+        final OpenAPIDeserializer deserializer = new OpenAPIDeserializer();
+        final SwaggerParseResult result = deserializer.deserialize(rootNode);
+
+        Assert.assertNotNull(result);
+
+        final OpenAPI openAPI = result.getOpenAPI();
+        Assert.assertNotNull(openAPI);
+
+        assertEquals(openAPI.getServers().get(0).getUrl(),"/");
```

Comment:
This seems right, based on https://github.com/OAI/OpenAPI-Specification/blob/3.0.0-rc2/versions/3.0.md#oasServers.